app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -199,7 +219,8 @@ class ConversationViewModel @Inject constructor(
                 ),
                 messageSource = MessageSource.CurrentUser,
                 messageHeader = MessageHeader(
-                    "Cool User",
+                    // TODO: Designs for deleted users?
+                    sender?.name ?: "Deleted User",
```

Comment:
On iOS we don't completely delete deleted users they are merely marked as deleted and we keep their name if we already have it.